doc/api/vm.md

```diff
@@ -439,9 +439,9 @@ let code =
  vm.runInThisContext(code)(require);
  ```
 
-*Note*: The `require()` in the above case shares the state with context it is
-passed from. This may introduce risks when untrusted code is executed, e.g.
-altering objects from the calling thread's context in unwanted ways.
+*Note*: The `require()` in the above case shares the state with the context it
+is passed from. This may introduce risks when untrusted code is executed, e.g.
+altering objects from the context in unwanted ways.
```

Comment:
I didn't change this sentence, but I think the author meant that `require` had been passed from the other context. However, it would be nice to reword the whole sentence, since there are some other quirks; e.g., why "shares state with the context" if it just shares the same context.